apps/system/js/scheduler.js

```diff
@@ -0,0 +1,63 @@
+/* global BaseModule */
+'use strict';
+
+(function() {
+  var Scheduler = function() {
+  };
+  Scheduler.SERVICES = [
+    'schedule'
+  ];
+  BaseModule.create(Scheduler, {
+    name: 'Scheduler',
+    DEBUG: false,
+    _start: function() {
+      this._queue = [];
+      this.hold(); // Hold on start until the parent tells us to release.
```

Comment:
The only way to achieve is an event to notify the scheduler to release; only launcher knows the timing and only scheduler knows when it is started.